microsoftgraph/microsoft-graph-explorer-v4
# Fix: Permissions styling

## Overview
Fixes #2440 

### Demo

[screen-recorder-wed-mar-08-2023-12-27-47.webm](https://user-images.githubusercontent.com/45680252/223727344-e1325244-96c5-4769-adf9-1c62dcdb4d97.webm)

[screen-recorder-wed-mar-08-2023-16-32-28.webm](https://user-images.githubusercontent.com/45680252/223727359-3a7750de-33f1-497c-acbd-bb1db60acd87.webm)

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Click on the Modify Permissions tab
Adjust your browser size and confirm that all items are aligned as per requirement
Sign in and out while interacting witht the tab
